docs/xml/elements/electoral_district.rst

```diff
@@ -1,24 +1,56 @@
 ElectoralDistrict
 =================
 
-.. todo::
+The ``ElectoralDistrict`` object represents the geographic area in which a single contest is held.
+Examples of ``ElectoralDistrict`` include: statewide, congressional district, a state senate
```

Comment:
Technically those are `DistrictType` values. I'd say examples would be "the state of Maryland", "Virginia's 5th Congressional District", or "Union School District".